system-tests/ft-suite/src/test/kotlin/com/hotels/styx/resiliency/OriginResourcesSpec.kt

```diff
@@ -67,7 +67,7 @@ class OriginResourcesSpec : StringSpec() {
             }
 
             // From the static configuration below
-            clientThreadCount shouldBe 3
+            clientThreadCount-initialThreadcount shouldBe 3
```

Comment:
We should probably remove this test.
It doesn't look very stable when running in shared JVM in parallel with others.

Reply:
you created this test for a regression a few months ago .so I would hesitate to just remove it... We are not running tests in parallel now, but I guess that if we did we might want to have a similar test where we can inject the pool-name (the prefix) or maybe verify this in a different way (e.g, exposing some reference to the thread pool and verifying it's the same).

Anyway, if we agree to remove it, it's fine for me!

Reply:
It would be nice to keep this test. But as it is at the moment it doesn't look stable. As you say, we would have to inject some kind of thread name prefix for this particular test. That could be a separate increment.